tools/nnyeah/nnyeah/FieldTransformations.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+namespace nnyeah {
+	public class FieldTransformations {
+		static Dictionary<string, Transformation> allTransforms;
+
+		public Dictionary<string, Transformation> GetTransforms (ModuleDefinition module)
+		{
+			if (allTransforms != null) {
+				return allTransforms;
+			}
+
+			allTransforms = new Dictionary<string, Transformation> ();
```

Comment:
We could have used a list init, not a huge problem, but would have saved you from typing.